test/fixtures/props/issueish-pane-props.js

```diff
@@ -34,9 +34,11 @@ export function issueishDetailControllerProps(opts, overrides = {}) {
     repositoryName: 'repository',
     ownerLogin: 'owner',
 
-    omitIssueish: false,
+    omitPullRequestData: false,
+    omitIssueData: false,
```

Comment:
Speaking as the one who put them there, I don't love these factories, btw 💔. I'm not sure what a decent alternative to constructing these complicated prop structures that Relay gives us is, though. More builders instead... ?

Reply:
yeah, this one is especially bad because to render an issue to test, you still need pull request data with a typename of `Issue`.  I also couldn't think of a better way to handle it, though.